src/routes/flavor.js

```diff
@@ -308,4 +315,69 @@ router.delete(
   })
 );
 
+/**
+ * GET Flavor Notes
+ * @query offset int
+ * @query limit int
+ */
+router.get(
+  '/:flavorId/notes',
+  authenticate(),
+  ensurePermission('flavors', 'read'),
```

Comment:
I can't recall, is there a separate subject for `flavor` singular?

Reply:
There is a `flavor`... I am wondering if these should use the `flavorNote`  subject though.

Reply:
Yeah, agreed that the new subject should be used.

Reply:
Updated these to flavorNote